common/lib/xmodule/xmodule/course_module.py

```diff
@@ -184,6 +184,11 @@ class CourseFields(object):
         help=_("Enter the date you want to advertise as the course start date, if this date is different from the set start date. To advertise the set start date, enter null."),
         scope=Scope.settings
     )
+    pre_requisite_courses = List(
```

Comment:
This is showing up in Studio Advanced Settings. 
![image](https://cloud.githubusercontent.com/assets/484484/5682786/97aa908c-97f0-11e4-9e50-bc2c808d4ecb.png)

You need to add this setting name to the list filtered out of Advanced Settings in course_metadata.py (FILTERED_LIST).